src/Post/Post.js

```diff
@@ -359,6 +364,85 @@ function CommentSection({
     </section>
   );
 }
+function MediaSection({ media_items, user_name }) {
+  const [mediaIndex, setMediaIndex] = React.useState(0);
+  const [videoPause, setVideoPause] = React.useState(false);
+  return (
+    <section className="img-container">
+      {media_items[mediaIndex].type === 'photo' ? (
+        <img
+          className="post-img"
+          src={media_items[mediaIndex].url}
+          alt={`${user_name}  Avatar`}
+          loading="lazy"
+        />
```

Comment:
Didn't we agree to create a component called `PostImage`? Because there will be tags and stuff inside the picture?